Zapper-fi/studio
# feat(maker): Convert to templates, fix key by including CDP ID

## Description

CDP positions are unique by their ID, not the ilk name.

## Checklist


## How to test?

demo.eth has 3 open CDPs against the USDC-A vault.
See https://oasis.app/owner/0xfc43f5f9dd45258b3aff31bdbe6561d97e8b71de#multiply
